docs/upgrade-1-8-summary.md

```diff
@@ -0,0 +1,68 @@
+```
+3:36PM INF Updated governance module minimum deposit from 1000000000000nhash to 50000000000000nhash
+3:36PM INF Updated attribute module max length value from 10000 to 1000
+3:36PM INF NOTICE: Starting migrations. This may take a significant amount of time to complete. Do not restart node.
+3:36PM INF Migrating Metadata Module from Version 2 to 3
+3:36PM INF Metadata step 1 of 7: Deleting bad indexes and getting good
+3:38PM INF Prefix 17: Done. Deleted 104354 entries, found 202006 good entries.
+3:38PM INF Prefix 18: Identifying good entries and deleting bad ones.
+3:39PM INF Prefix 18: Done. Deleted 52808 entries, found 101714 good entries.
+3:39PM INF Prefix 19: Identifying good entries and deleting bad ones.
+3:39PM INF Prefix 19: Done. Deleted 4 entries, found 6 good entries.
+3:39PM INF Prefix 20: Identifying good entries and deleting bad ones.
+3:39PM INF Prefix 20: Done. Deleted 2607 entries, found 3173 good entries.
+3:39PM INF Done identifying good indexes and deleting bad ones.
+3:39PM INF Metadata step 2 of 7: Reindexing scopes
+3:40PM INF Done reindexing 10449 scopes. All 112164 are now indexed.
+3:40PM INF Metadata step 3 of 7: Reindexing scope specs
+3:40PM INF Done reindexing 0 scope specs. All 6 are now indexed.
+3:40PM INF Metadata step 4 of 7: Reindexing contract specs
+3:40PM INF Done reindexing 2607 contract specs. All 5780 are now indexed.
+3:40PM INF Metadata step 5 of 7: Identifying sessions to keep
+3:48PM INF Done identifying a total of 237678 session ids used by 1171973 records.
+3:48PM INF Metadata step 6 of 7: Finding empty sessions
+3:55PM INF Done deleting 640843 empty sessions.
+3:55PM INF Finished Migrating Metadata Module from Version 2 to 3
+3:55PM INF Adding a 10 hash (10,000,000,000 nhash) msg fee to MsgBindNameRequest (1/6)
+3:55PM INF Adding a 100 hash (100,000,000,000 nhash) msg fee to MsgAddMarkerRequest (2/6)
+3:55PM INF Adding a 10 hash (10,000,000,000 nhash) msg fee to MsgAddAttributeRequest (3/6)
+3:55PM INF Adding a 10 hash (10,000,000,000 nhash) msg fee to MsgWriteScopeRequest (4/6)
+3:55PM INF Adding a 10 hash (10,000,000,000 nhash) msg fee to MsgP8EMemorializeContractRequest (5/6)
+3:55PM INF Adding a 100 hash (100,000,000,000 nhash) msg fee to MsgSubmitProposal (6/6)
+3:55PM INF Successfully upgraded to: green with version map: map[attribute:2 auth:2 authz:1 bank:2 capability:1 crisis:1 distribution:2 evidence:1 feegrant:1 genutil:1 gov:2 ibc:2 marker:2 metadata:3 mint:1 msgfees:1 name:2 params:1 slashing:2 staking:2 transfer:1 upgrade:1 vesting:1 wasm:1]
+```
+7. When the upgrade has finished it your node will wait for other peers and the validators to complete the upgrade as well.  The active voting power must reach the 2/3 majority threshold for the network to continue.
+
+
+# What to do if something goes wrong
+
+First off, make sure you have in contact with the other network participants on the Provenance discord server. https://discord.gg/HszDgS6R
```

Comment:
Maybe have the link as a md link structure?

Reply:
I thought about that ... but I wanted the full raw link to show here because it makes it clear its an invite link...